src/amo/components/AddonHead/index.js

```diff
@@ -6,6 +6,7 @@ import Helmet from 'react-helmet';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
+import HrefLang from 'amo/components/HrefLang';
```

Comment:
`HrefLang` doesn't immediately convey what this component does. How about `AltLangLinks`? Or something.

Reply:
`AlternateLinks`?

Reply:
I like `AlternateLinks`